src/Concerns/RegistersExceptionHandlers.php

```diff
@@ -16,20 +16,21 @@ trait RegistersExceptionHandlers
     /**
      * Throw an HttpException with the given data.
      *
-     * @param  int  $code
+     * @param  int  $statusCode
      * @param  string  $message
      * @param  array  $headers
+     * @param  int  $code
      * @return void
      *
      * @throws \Symfony\Component\HttpKernel\Exception\HttpException
      */
-    public function abort($code, $message = '', array $headers = [])
+    public function abort($statusCode, $message = '', array $headers = [] , int $code = 0)
```

Comment:
Renaming parameter names is a breaking change for PHP 8 as well as is the extra parameter addition. Please send this to master.